test/testSimplifier.ts

```diff
@@ -162,6 +162,9 @@ function tokenizeKeySequence(sequence: string): string[] {
 
     if (char === '>') {
       isBracketedKey = false;
+      if (key.startsWith("<c-")) {
```

Comment:
Ahh, I understand now.

Here's the thing. We have to have this hack because currently we're inconsistently handling ctrl keys between our tests and our infrastructure. We should be using &lt;c-*&gt; everywhere. 

As insane as it sounds, I'd actually prefer you to NOT fix it here and instead make the hack bigger, so that it becomes an increasingly large eye-sore which will cause people (mostly me) to want to fix it the correct way.